primer-io/primer-sdk-ios
# fix: Start loading spinner earlier in Drop-in Card Form

# Description

[CHKT-2292](https://primerapi.atlassian.net/browse/CHKT-2292)

- This change starts the loading spinner slightly sooner when making a payment with the Drop-in card form and removes the delay experienced by the user.

# Manual Testing

Make a card payment and observe the spinner starts immediately upon tapping the Pay button.

# Contributor Checklist


# Reviewer Checklist


# Before Merging


# Other Stuff

- You can find out more about our automation checks [here](https://primerio.notion.site/iOS-Automation-Checks-198a1eb0e8994d999fb696d5902d97bb)
- Find out more about conventional commits [here](https://primerio.notion.site/Conventional-Commits-6ecfd6a0269a4db2af76d9f0537936b3)

[CHKT-2292]: https://primerapi.atlassian.net/browse/CHKT-2292?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
